x/ibc-dns/server/types/packet.go

```diff
@@ -39,8 +41,8 @@ func (p RegisterDomainPacketData) GetBytes() []byte {
 	return bz
 }
 
-func (p RegisterDomainPacketData) GetTimeoutHeight() uint64 {
-	return math.MaxUint64
+func (p RegisterDomainPacketData) GetTimeoutHeight() ibcclienttypes.Height {
+	return ibcclienttypes.NewHeight(0, math.MaxInt64)
```

Comment:
If specify MaxUint64, there is a possibility of overflow with the following codes
https://github.com/cosmos/cosmos-sdk/blob/master/x/ibc/core/02-client/types/height.go#L61

Reply:
I've fixed it at [this commit](https://github.com/cosmos/cosmos-sdk/pull/7944).
After a new version that includes this commit is released, we should revert the changes.